ui/src/masking.ts

```diff
@@ -60,19 +60,24 @@ export async function callWithMaskHandling<T>(callback: () => Promise<T>, valueT
 
 /**
  * Best effort to mask all data that could potentially identify a user
+ * @param lessAggressive If set to true, the most aggressive masking will be skipped
```

Comment:
nit: maybe include a little bit more detail here, i.e. "skipping email and id if true" or something in those lines. lessAggressive and most aggressive read a little vague.

Reply:
Well, I want to keep it a big vague so we don't have to update the comment every time the logic changes